src/Components/SplashScreen/index.js

```diff
@@ -19,21 +19,31 @@ const container = {
 };
 
 const SplashScreen = () => {
+  const [showConfetti, setShowConfetti] = useState(false);
+
   useEffect(() => {
```

Comment:
# Code Refactor Request

Hi! This is a great piece of work and a truly valuable contribution. Thank you for your efforts on this! We just have a small request for some changes in your current Pull Request.

Your code is solid, and there are no changes needed for the logic itself. We did, however, notice that you’ve used the `useEffect` hook in the `index.js` file. In our current project structure, we typically place all custom hooks (including `useEffect`) in a separate file, specifically the `useSplashScreen.js` file.

You can find this file here in the directory: [src/Components/SplashScreen](https://github.com/Dtech-Dbug/MemeTem/tree/main/src/Components/SplashScreen).

Could you kindly move all your `useEffect` hooks and any other custom hooks into the `useSplashScreen.js` file? Once done, you can simply import it into `index.js` for usage and implementation.

The reason we're suggesting this refactor is to improve code readability and make it easier for others to understand the structure. It will also help us with any future debugging or error handling, should issues arise later on.

Thanks again for your hard work! We really appreciate your contribution and look forward to your updated pull request.